website/docs/integrations/trello.mdx

```diff
@@ -4,23 +4,67 @@ title: Trello Power-Up - Manage feature flags from Trello
 description: ConfigCat Trello Power-Up. This is a step-by-step guide on how to connect and manage feature flags from Trello boards and cards.
 ---
 
-Turn features On / Off right from a linked Card on your Trello board with <a href="https://trello.com/power-ups/5e694b66d2511a3601ebd0fb" target="_blank">ConfigCat's Trello Power-Up</a>. Also you can add Targeting or Percentage Rules just as easily.
+The <a href="https://trello.com/power-ups/5e694b66d2511a3601ebd0fb" target="_blank">ConfigCat's Trello Power-Up</a> allows you to connect your Trello Cards and feature flags. Create or link existing flags to your Trello cards without leaving your Trello board.
 
-## Installation
+Turn features On/Off right from a linked card on your Trello board. You can also easily modify the linked flags to edit or add new Targeting or Percentage Rules.
```

Comment:
Ez is egy értékajánlat, itt is azt javaslom, hogy ne az Installation szekcióban legyen, hanem a doksumentum elején az első egy-két mondatban.

Reply:
A doksi elején van. Pont az installation szekció került hátrább.